libclambcc/ClamBCVerifier/ClamBCVerifier.cpp

```diff
@@ -41,23 +41,30 @@
  * }
  */
 
+
+#include "Common/ClamBCDiagnostics.h"
+#include "Common/ClamBCModule.h"
+#include "Common/clambc.h"
+#include "Common/ClamBCUtilities.h"
+
+
+
+
 #include <llvm/Pass.h>
-#include "llvm/IR/Function.h"
-#include "llvm/Support/raw_ostream.h"
+#include <llvm/IR/Function.h>
```

Comment:
Good call switching from `"` to `< >` 👍

Reply:
And having them mixed was driving me crazy.